src/inabox/inabox-viewport.js

```diff
@@ -322,7 +322,11 @@ export class ViewportBindingInabox {
 
   /** @private */
   remeasureAllElements_() {
-    this.getChildResources().forEach(resource => resource.measure());
+    const resources = this.getChildResources();
+    // Force remeasurements.
+    resources.schedulePass(0, /* opt_relayoutAll */ true);
```

Comment:
Isn't `resources` an array? Let's make sure to run this code path manually.

Reply:
Ha, I got confused. Fixed.